build-scripts/config_common.cmake

```diff
@@ -212,6 +215,7 @@ endif ()
 message ("-- Build Configurations:")
 message ("     Build as target ${WAMR_BUILD_TARGET}")
 message ("     CMAKE_BUILD_TYPE " ${CMAKE_BUILD_TYPE})
+message ("     BUILD_SHARED_LIBS " ${BUILD_SHARED_LIBS})
```

Comment:
Would u mind adding one more message about `WAMR_BUILD_PLATFORM` between L216 and L217. like:

``` cmake
message (" Build for platform ${WAMR_BUILD_PLATFORM}")
```